app/models/pipeline_run.rb

```diff
@@ -570,6 +570,23 @@ def nonhost_fastq_s3_paths
     files
   end
 
+  def betacoronavirus_fastq_s3_paths
+    # must be in sync with naming in idseq_dag/steps/nonhost_fastq.py
+    s3_path = lambda do |name|
+      "#{postprocess_output_s3_path}/betacoronavirus__#{name}"
+    end
+
+    s3_paths = [
+      s3_path.call(sample.input_files[0].without_gz),
+    ]
+
+    if sample.input_files.length == 2
+      s3_paths << s3_path.call(sample.input_files[1].without_gz)
```

Comment:
Is it necessary to name this file after the user-provided input file? It seems like a complexity we could get rid of. We've also noted that for security purposes it would be better to use our own file names in our backend rather than user-provided ones and this would be a good opportunity to get into that habit. cc @davidrissato ?

Reply:
Although it would be a good idea to use our own file names, I believe in this case he might have done that on purpose, since bulk downloads can download multiple samples at once, and it would be hard for users to identify based on the sample id. Am I correct?

Reply:
Bulk downloads has a step to rename the files, and includes the sample name and sample id in the file name. So it's okay to give these files a static name like betacoronavirus_R1.fastq if we want.

Reply:
It's not necessary at all. I just followed the example of the previous PR which I presumed was done to convey more context to the downloader. 

It's good to learn that we prefer to use our own file names. The bulk download code renames the files according to the sample name which is derived from the original uploaded file name. So is that  a security risk as well?

Reply:
> @MarkAZhang said:
> Bulk downloads has a step to rename the files, and includes the sample name and sample id in the file name
By "sample name" do you mean `Sample#name` or `InputFile#name`? Could you point us to this code?

Reply:
Yeah, it's in get_output_file_prefix. It's the sample name.

Reply:
Considering we want using the sample name, I recommend being very aggressive on name cleanups. My suggestion is only allowing `/[0-9a-zA-Z_]/`. Anything else should be replaced by an underscore.